src/app/recipes/recipe-list/recipe-list.component.ts

```diff
@@ -27,14 +28,16 @@ export class RecipeListComponent implements OnInit, OnDestroy {
 
   ngOnInit() {
     this.RecipeSubscription = this.store
-      .pipe(
-        select('recipes'),
-        map((recipesState) => {
-          return recipesState.recipes;
-        })
-      )
-      .subscribe((recipes: Recipe[]) => {
-        this.recipes = recipes;
+      .pipe(select('recipes'))
+      .subscribe((recipesState) => {
+        this.recipes = recipesState.recipes;
+        this.recipesNumber = recipesState.maxRecipes;
+        this.recipesOnPage = recipesState.recipesOnPage;
```

Comment:
Please use specific selectors

Reply:
added specific selectors and pipe `async`